app/docs/0.11.x/configuration.md

```diff
@@ -589,99 +582,62 @@ name                            | description
 
 ---
 
-#### Clustering section
-
-In addition to pointing to the same database, each Kong node must join the
-same cluster.
-
-Kong's clustering works on the IP layer (hostnames are not supported, only
-IPs) and expects a flat network topology without any NAT between the
-datacenters.
-
-A common pattern is to create a VPN between the two datacenters such that
-the flat network assumption is not violated.
-
-See the clustering reference for more informations:
-https://getkong.org/docs/latest/clustering/
-
----
-
-##### **cluster_listen**
-
-Address and port used to communicate with other nodes in the cluster.
-All other Kong nodes in the same cluster must be able to communicate over both
-TCP and UDP on this address. Only IPv4 addresses are supported.
-
-Default: `0.0.0.0:7946`
-
----
-
-##### **cluster_listen_rpc**
-
-Address and port used to communicate with the cluster through the agent
-running on this node. Only contains TCP traffic local to this node.
-
-Default: `127.0.0.1:7373`
-
----
-
-##### **cluster_advertise**
-
-By default, the `cluster_listen` address is advertised over the cluster.
-If the `cluster_listen` host is '0.0.0.0', then the first local, non-loopback
-IPv4 address will be advertised to other nodes.
-However, in some cases (specifically NAT traversal), there may be a routable
-address that cannot be bound to. This flag enables advertising a different
-address to support this.
-
-Default: none
-
----
-
-##### **cluster_encrypt_key**
+#### Datastore cache section
 
-Base64-encoded 16-bytes key to encrypt cluster traffic with.
+In order to avoid unecessary communication with the datastore, Kong caches
+entities (such as APIs, Consumers, Credentials...) for a configurable period
+of time. It also handles invalidations if such an entity is updated.
```

Comment:
perhaps a brief note on the pros/cons of a shorter or longer db_cache_ttl value?